notebooks/simple-feature-transformations-explain-local.ipynb

```diff
@@ -307,7 +307,7 @@
     "# LGBMExplainableModel can be replaced with LinearExplainableModel, SGDExplainableModel, or DecisionTreeExplainableModel\n",
     "# explainer = MimicExplainer(clf.steps[-1][1], \n",
     "#                            x_train, \n",
-    "#                            LGBMExplainableModel, \n",
+    "#                            DecisionTreeExplainableModel, \n",
```

Comment:
why using DecisionTreeExplainableModel?

Reply:
jupyter notebook autosave....
I'll fix

Reply:
More info: I was testing all the notebooks and all versions of the models and jupyter notebook autosaved the notebook. Because the different models require hand editing, I manually commented out the MimicExplainer and needed to manually change the model back too. I fixed it now. checking it in for the PR soon.

Reply:
fixed